<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="backgroud" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>811</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="inside">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>0</width>
      <height>391</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>0</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Iniciar sesión&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-radius:0px;
border-bottom-color: rgba(46,82,101,200);
color: rgb(0,0,0);
padding-bottom: 7px;
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="continueB">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Continue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>Continuar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ver">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>137</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>show.png</normaloff>show.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="cover">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #AAAAFF, stop:1 #ffaaff);

border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;

font: 75 20pt &quot;Script MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;Bake Kepeer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>271</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url('./resources/cake.png');
background-color: transparent;
background-repeat: no-repeat;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab1" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>75</y>
      <width>41</width>
      <height>0</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <widget class="QPushButton" name="closeApp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed {
       background-color: rgba(0, 0, 0, 0.5);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>cross.png</normaloff>cross.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>75</y>
      <width>41</width>
      <height>0</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>300</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <widget class="QPushButton" name="minApp">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>8</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed {
       background-color: rgba(0, 0, 0, 0.5);
 }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>blackLine.png</normaloff>blackLine.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <zorder>tab1</zorder>
   <zorder>inside</zorder>
   <zorder>cover</zorder>
   <zorder>tab2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="fondo.qrc"/>
 </resources>
 <connections/>
</ui>
